### DISCUSSION TITLE
Considering the Hailo8, anybody got it working on an Intel system?

### DISCUSSION THREAD

**Discussion by aeozyalcin** [blakeblackshear/frigate#15258] [Ask A Question] [answered]
As the title says, I am considering the [Waveshare Hailo8 M.2 PCIE module](https://www.waveshare.com/hailo-8.htm), to run on my Intel N100 Mini PC. I see that Frigate v15 has Hailo8l support, but it's not clear if it's only for Raspberry Pi's, or if it would also work on x86 system. Has anyone gotten the Hailo8/Hailo8L working on an x86 system? If so, would you recommend it?

**Comment by NickM-27** [comment, marked as answer]
Yes, amd64 systems are supported
